android/expoview/src/main/java/host/exp/exponent/views/ExponentImageButton.kt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+package host.exp.exponent.views
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.MotionEvent
+import androidx.appcompat.widget.AppCompatImageButton
+
+class ExponentImageButton : AppCompatImageButton {
```

Comment:
Is this switch to AppCompatImageButton safe?

Reply:
Yep, verified that the image buttons in the error activity/fragments still work (the only place this is used).